src/clx.js

```diff
@@ -0,0 +1,35 @@
+import { cache } from './core/hash';
+import { update } from './core/update';
+import { toHash } from './core/to-hash';
+import { getSheet } from './core/get-sheet';
+
+/**
+ * clx
+ * @param {...String} classes
+ */
+function clx() {
+    let ctx = this || {};
+    let classes = [].reduce.call(arguments, (classes, c) => classes.concat(c.split(' ')), []);
+
+    let id = toHash(classes.join(''));
+
+    let hashedClasses = classes.reduce((hashedClasses, c) => {
```

Comment:
This used to be structured differently, but i dont think this needs to be a reduce anymore. Map will be smaller and more efficient 💯